flight/targets/sprf3e/bl/pios_board.c

```diff
@@ -0,0 +1,88 @@
+/**
+ ******************************************************************************
+ * @addtogroup Bootloader Bootloaders
+ * @{
+ * @addtogroup spf3e SP Racing F3E support files
```

Comment:
SVGs have to be created for the new board. It's on the todo (see initial post). Seppuku SVGs are only a placeholder.
Board picture is already exchanged by my own photo from my own board. No license problems.